script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -287,42 +306,47 @@ I extend the blade of my knife.
 [sha:73a60ad43eaab847d3bbedcf7cbbb8cff4f807dc]{
 -- Page 267, Offset 33955.
 -- 瞬間。
-In an instant.
+The next instant, she's gone.
 }
 [sha:531d904fdce4b6399d26821c007e7abb2b0ae249]{
 -- Page 267, Offset 33956.
 -- 先輩の体が<爆|は>ぜた。
-Senpai's body bursts forth.
+// Assuming this is bursts (and disappears)? Seems weird
+Senpai's body has winked out of existence.
```

Comment:
I agree with the pattern, but I don't think bursts forth is correct for 1. - IIRC she does disappear in game here. Bursts forth would be more like makes a head-on rush for him, but instead I think it's more like
1. She fuckin disappeard
2. Wait, no she didn't, she just
3. got real low to the ground as she was approaching

Reply:
I thought it was something like, "in an instant, Senpai's [body] bursts forth", but I guess I shouldn't put it past Nasu to use it in some kind of "explodes and disappears" way? Honestly, I could go either way after thinking about it, but the implication feels like he's just "oh shit, she flashstepped", "actually no she just did some weird karate shit." I dunno.

Reply:
Yeah I feel like it needs to be "something impossible??? oh" and bursting forth is the possible/expected action, so it sits weirdly with the pattern

Reply:
Well, it does bother me still, looking at it, because in no way do I think 爆ぜる is used like this? But I'm not sure about the alternative. It's understandable enough. Maybe it's best postponed to GPR.

Reply:
Yeah it's definitely weird. The in-context examples I can find are mostly to do with the crackle of things burning.